rest-api/test/unit_test/offline_test/biobank_samples_pipeline_test.py

```diff
@@ -137,12 +136,24 @@ def test_sample_from_row_invalid(self):
     with self.assertRaises(biobank_samples_pipeline.DataError):
       biobank_samples_pipeline._create_sample_from_row(row, get_biobank_id_prefix())
 
+  def test_sample_from_row_old_test(self):
```

Comment:
As noted offline: we need to call _upsert_samples_from_csv() instead. This test doesn't necessarily check anything we care about.

In the interests of time we can merge without a test this time and do a manual test (we're going to want to do that anyway.) But let's get this fixed tomorrow.